### ISSUE TITLE
Function with default parameters disables suggestions, red error squiggles etc.

### ISSUE THREAD

**Issue by rebornplusplus** [microsoft/vscode-cpptools#5405]
**Type: LanguageService**

**Describe the bug**
- OS and Version: Ubuntu 18.04.4 LTS
- VS Code Version: 1.43.2
- C/C++ Extension Version: 0.28.0-insiders
- Other extensions you installed (and if the issue persists after disabling them):
- Does this issue involve using SSH remote to run the extension on a remote machine?: No
- A clear and concise description of what the bug is, including information about the workspace (i.e. is the workspace a single project or multiple projects, size of the project, etc).

I had this code:
```cpp
#include <bits/stdc++.h>

using namespace std;

int square(int p) {
	return p * p;
}

int main() {
	cout << square(4) << "\n";
	cout << 2p; // error squiggles

	return 0;
}
```
Upon clicking Ctrl + Space, the following suggestions appear and the error squiggles work fine too.
![image](https://user-images.githubusercontent.com/18555205/80899457-2d882600-8d32-11ea-8683-49a13b4fee9c.png)

However, when I decide the parameter `p` in `square` to be a default one, the suggestions only show the C++ keywords and no error squiggles or whatsoever. All that's left are the keywords.
![image](https://user-images.githubusercontent.com/18555205/80899429-cd917f80-8d31-11ea-957c-42cc88894817.png)

**Steps to reproduce**

1. Type the following code.
```cpp
#include <bits/stdc++.h>

using namespace std;

int square(int p = 2) {
	return p * p;
}

int main() {
	cout << square(4) << "\n";
	cout << 2p; // should show error squiggles

	return 0;
}
// Check with Ctrl + Space
```

**Expected behavior**

I want those autocomplete suggestions, those error squiggles.

<details>
  <summary><strong>Logs</strong></summary>

```
-------- Diagnostics - 5/3/2020, 11:53:11 AM
Version: 0.28.0-insiders
Current Configuration:
{
    "name": "Linux",
    "includePath": [
        "${workspaceFolder}/**"
    ],
    "defines": [],
    "compilerPath": "/usr/bin/gcc",
    "cStandard": "c11",
    "cppStandard": "c++17",
    "intelliSenseMode": "clang-x64",
    "compilerArgs": [],
    "browse": {
        "path": [
            "${workspaceFolder}/**"
        ],
        "limitSymbolsToIncludedHeaders": true
    }
}
Translation Unit Mappings:
[ /home/rough/codir/geo/temp.cc ]:
    /home/rough/codir/geo/temp.cc
Translation Unit Configurations:
[ /home/rough/codir/geo/temp.cc ]: not ready
    Process ID: 7993
    Memory Usage: 0 MB
    Compiler Path: /usr/bin/gcc
    Includes:
        /usr/include/c++/7
        /usr/include/x86_64-linux-gnu/c++/7
        /usr/include/c++/7/backward
        /usr/lib/gcc/x86_64-linux-gnu/7/include
        /usr/local/include
        /usr/lib/gcc/x86_64-linux-gnu/7/include-fixed
        /usr/include/x86_64-linux-gnu
        /usr/include
    Standard Version: c++17
    IntelliSense Mode: gcc-x64
    Other Flags:
        --g++
        --gnu_version=70500
Total Memory Usage: 0 MB
```
</details>

**Screenshots**

**Additional context**

**Comment by sean-mcmanus** [comment]
This is already fixed in our pending 0.28.0-insiders2 update, same root cause as https://github.com/microsoft/vscode-cpptools/issues/5326 . We're planning to release it later today.

**Comment by sean-mcmanus** [comment]
Fixed with https://github.com/microsoft/vscode-cpptools/releases/tag/0.28.0-insiders2

**Comment by elahehrashedi** [comment]
This issue is fixed in 0.28.0.
0.28.0 release: https://github.com/microsoft/vscode-cpptools/releases/tag/0.28.0
